app/controllers/frontend_log_controller.rb

```diff
@@ -34,7 +38,11 @@ class FrontendLogController < ApplicationController
   # rubocop:enable Layout/LineLength
 
   def create
-    frontend_logger.track_event(log_params[:event], log_params[:payload].to_h)
+    if error_event?
+      NewRelic::Agent.notice_error(FrontendError.new, custom_params: log_params[:payload].to_h)
+    else
+      frontend_logger.track_event(log_params[:event], log_params[:payload].to_h)
+    end
```

Comment:
what if we brought back the "symbol or proc" logic we used to have before #7110 in `EVENT_MAP` and then supply a proc for this event?

Reply:
> what if we brought back the "symbol or proc" logic we used to have before #7110 in `EVENT_MAP` and then supply a proc for this event?

Ooh, that's a very interesting idea! Let me give that a shot.

Reply:
I had fun and gave it a shot here https://github.com/18F/identity-idp/pull/9117